client/state/action-types.js

```diff
@@ -212,6 +212,7 @@ export const EDITOR_SAVE = 'EDITOR_SAVE';
 export const EDITOR_EDIT_RAW_CONTENT = 'EDITOR_EDIT_RAW_CONTENT';
 export const EDITOR_INIT_RAW_CONTENT = 'EDITOR_INIT_RAW_CONTENT';
 export const EDITOR_RESET_RAW_CONTENT = 'EDITOR_RESET_RAW_CONTENT';
+export const EDITOR_SET_GUTENBERG = 'EDITOR_SET_GUTENBERG';
```

Comment:
we might want to double-check our naming here. if we are going to offer opt-out then something like `EDITOR_SELECT` or `EDITOR_OPT_IN` which takes a parameter specifying the editor experience might be more clear in its intent